website/src/theme/MDXComponents/index.js

```diff
@@ -1,85 +1,32 @@
-/**
- * Copyright (c) Facebook, Inc. and its affiliates.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE file in the root directory of this source tree.
- */
-import React, { isValidElement } from 'react'
-import Head from '@docusaurus/Head'
-import Link from '@docusaurus/Link'
-import CodeBlock from '@theme/CodeBlock'
-import Heading from '@theme/Heading'
-import Details from '@theme/Details'
-
-// NOTE: This copy gives us the posibility to define our own scopes.
-// Keep in mind that this class can have fast breaking changes from updates. so maybe swizzle it again via yarn swizzleMDX
-//
-// https://github.com/facebook/docusaurus/issues/2152#issuecomment-597558963
-import VideoContainer from '../VideoContainer'
-
-// MDX elements are wrapped through the MDX pragma
-// In some cases (notably usage with Head/Helmet) we need to unwrap those elements.
-function unwrapMDXElement(element) {
-  if (element?.props?.mdxType && element?.props?.originalType) {
-    const { mdxType, originalType, ...newProps } = element.props
-    return React.createElement(element.props.originalType, newProps)
-  }
-
-  return element
-}
+import React from 'react';
+import MDXHead from '@theme/MDXComponents/Head';
+import MDXCode from '@theme/MDXComponents/Code';
+import MDXA from '@theme/MDXComponents/A';
+import MDXPre from '@theme/MDXComponents/Pre';
+import MDXDetails from '@theme/MDXComponents/Details';
+import MDXHeading from '@theme/MDXComponents/Heading';
```

Comment:
whats the difference between `Heading` and `MDXHeading`? `Heading` still used? if not, can it then be deleted? same for the other files